redis_inventory.py

```diff
@@ -58,12 +58,7 @@ def put_product(self, id, info):
 
 
   def delete_product(self, product_id):
-    if self.redis.exists(product_id):
-      self.redis.delete(product_id)
-      return True
-    else:
-      return False
-
+    self.redis.delete(product_id)
```

Comment:
Thanks! From the lecture, Delete should only return 204 No Content even if the resource wasn't found because Delete is ***idempotent***.